docs/adr/0002-tarball-hosting.md

```diff
@@ -0,0 +1,86 @@
+# 1) Use Github for hosting release tarballs {#adr_0002}
+
+Date: 2020-11-20
+
+## Status
+
+proposed
+
+## Context
+
+We need to determine a place to host our new release artifacts (tarballs). 
```

Comment:
If you’re asking me to review this ADR then I need to know a bit more about what these artifacts / tarballs are. Is it one tarball per release or several? What is included in these artifacts? If there are several tarballs per release, how does the Wikibase tarball differ from what you can download from ExtensionDistributor today?

Reply:
Yes, sorry about just adding you without the full context.

This ADR will only concern where we are supposed to host "the release tarball(s)" (ie things the release pipeline produces)

**Is it one tarball per release or several**

This was discussed a bit yesterday and the up until then we were envisioning individual tarballs for each component(Wikibase, extensions that support it) but some good arguments for something more along the lines of a "Wikibase bundle" was presented. WDQS, WDQS-UI would be separated as they are not extensions to mediawiki. The final composition of this is still up for debate.

**If there are several tarballs per release, how does the Wikibase tarball differ from what you can download from ExtensionDistributor today**

This is one of the arguments I see for actually putting everything in one bundle/tar. Otherwise the only difference would be whats described here (https://github.com/wmde/wikibase-release-prototype/pull/10) (Will aim to bring this idea of bundling everything up for discussion again today)